.vscode/settings.json

```diff
@@ -39,12 +39,6 @@
     "twentyhq"
   ],
   "typescript.preferences.importModuleSpecifier": "non-relative",
-  "[javascript][typescript][typescriptreact]": {
-    "editor.codeActionsOnSave": {
-      "source.fixAll.eslint": "explicit",
-      "source.addMissingImports": "always"
```

Comment:
source.addMissingImports was already set in previous `typescriptreact` array.
Removing it here solves the issue and imports still work